components/molecule/thumbnail/src/index.scss

```diff
@@ -0,0 +1,97 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$sizes-sui-molecule-thumbnail: (
```

Comment:
Ad far as i know, SCSS vars should not have `sui` prefix:

https://github.schibsted.io/scmspain/frontend-all-wiki/wiki/Naming-of-Components%27-Sass-Variables

Reply:
Naming should follow the convention, yes

Reply:
ok